src/housing/processing/processing.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+import numpy as np
+import pandas as pd
+from sklearn.compose import ColumnTransformer
+from sklearn.impute import SimpleImputer
+
+logger = logging.getLogger()
```

Comment:
`logging.getLogger(__name__)` would ensure that the module name is included as part of the log msg. useful for debugging.